ksqldb-engine/src/main/java/io/confluent/ksql/services/ConnectJsonMapper.java

```diff
@@ -40,14 +41,15 @@
 
   final ObjectMapper buildMapper() {
     final SimpleModule module = new SimpleModule();
-    module.addSerializer(PluginInfo.class, new PluginInfoJsonSerializer());
+    module.addSerializer(SimpleConnectorPluginInfo.class, new PluginInfoJsonSerializer());
 
     return new ObjectMapper()
         .disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET)
         .registerModule(new Jdk8Module())
         .registerModule(new StructSerializationModule())
         .registerModule(new KsqlTypesSerializationModule())
         .registerModule(module)
+        .registerModule(new GuavaModule())
```

Comment:
Why is this module being added? It looks like the two public-facing classes (SimpleConnectorInfo and SimpleConnectorPluginInfo) aren't using guava's immutable types, and that the new classes that do use guava's immutable types are internal-only, but if that's the case then I'm not seeing why the new types that do use guava'a immutable types are being serialized/deserialized by this mapper. What am I missing?